whisper/whisperv6/peer.go

```diff
@@ -130,7 +130,7 @@ func (peer *Peer) handshake() error {
 		}
 	}
 
-	isRemotePeerLightNode, err := s.Bool()
+	isRemotePeerLightNode, _ := s.Bool()
```

Comment:
Shouldn't we rather handle this, than ignore? Ping @gballet

Reply:
this is a feature required by status, I don't think Parity has it. So if that field is not present, then `isRemotePeerLightNode == false` should be assumed.